tmt/steps/execute/internal.py

```diff
@@ -354,7 +358,8 @@ def execute(
         remote_command = ShellScript(TEST_WRAPPER_TEMPLATE.render(
             INTERACTIVE=self.data.interactive,
             TTY=test.tty,
-            REMOTE_COMMAND=ShellScript(command)
+            REMOTE_COMMAND=ShellScript(command),
+            SCRIPTS_DEST_DIR=guest.scripts_path
```

Comment:
How about `GUEST_SCRIPTS_PATH`? We already have a name for that value, `guest.scripts_path`, it would be nice to stick to it

Reply:
ack

Reply:
should be resolved